docs/_includes/projects/properties/declare/my-element.js

```diff
@@ -1,30 +1,25 @@
-import { LitElement, html } from 'lit-element';
+import {LitElement, html} from 'lit-element';
 
 class MyElement extends LitElement {
-  static get properties() { return {
-    prop1: { type: String },
-    prop2: { type: Number },
-    prop3: { type: Boolean },
-    prop4: { type: Array },
-    prop5: { type: Object }
-  };}
+  static get properties() {
+    return {
+      greeting: {type: String},
+      data: {attribute: false},
+      items: {},
```

Comment:
Like below I think we want either `attribute: false` or `type: Array`

Reply:
👍